content/blog/2020-08-14-spreadsheet-workflows-using-r/index.Rmarkdown

```diff
@@ -0,0 +1,242 @@
+---
+title: Spreadsheet Workflows in R
+date: '2020-08-14'
+slug: spreadsheets-using-r
+categories:
+  - learn
+tags:
+  - spreadsheets
+  - tidyverse
+author:
+  - Devin Johnson
+photo:
+  url: https://unsplash.com/photos/Wpnoqo2plFA
+  author: Mika Baumeister
+---
+
+There's so much to discover and learn when working with spreadsheets. As a summer intern at RStudio my project has focused on creating a resource that lies at the intersection between spreadsheets and R, aimed at users who either exclusively use one or the other or whose work lives in the intersection of the two. This project has focused on a plethora of topics from comparing numerous R package functionalities that work with spreadsheets, to writing about tips and tricks on best practices for working in both a spreadsheet software and in RStudio. 
+
+In this post I'd like to share a part of this work, specifically focused on the R versions of common spreadsheet workflows. To do that we'll assume the role of a data analyst in two hypothetical scenarios and walk through the tidyverse equivalent of common analyses and tasks one is likely to come across when working with data stored in spreadsheets.   
+
+For our first story, it's day one as a new data analyst and our boss has asked us to take a look at the company's finances.
+They send you a spreadsheet, available in an Excel workbook, and ask you to analyze spending patterns over the current year and give your assessment into where spending is going overboard.
+
+We've decided to run our analysis in RStudio to take advantage of data wrangling and visualization functions from the tidyverse as well as to create a reproducible workflow in case we want to revisit this analysis later.  First, let's load in our data and take a look at it in RStudio. 
+
+If you would like to follow along, you can download the Excel file called `spending.xlsx` [here](https://drive.google.com/file/d/1_euqWVOt2lx79POl0upk5oQKXG_70dsS/view?usp=sharing) by clicking on the Download icon (arrow pointing down) on the top right corner. 
+Note that in the code chunk below we are reading this file from a folder called `data` so we recommend you create such a folder in your working directory and place the file you downloaded there. 
+You can also follow along with the analysis on RStudio Cloud [here](https://rstudio.cloud/project/1523407).
+
+```{r,message=FALSE}
+library(tidyverse)
+library(readxl)
+
+spending <- read_excel("data/spending.xlsx")
+spending
+```
+
+We take note of the month column as well as amount spent, budget, and, expense type. We'll hold off on looking at the column `num_of_expenses`, it may prove useful down the line but not pressing for a quick look into the data.
+
+
+## Nested logic
```

Comment:
I really like where this example has gone.

I think it's more powerful to the "spreadsheets --> R" person if the spreadsheet version comes *first*, i.e. "here's what you would usually do in Excel and here's the equivalent in R". I think that also creates a nicer ending to the section, as a side effect.

Reply:
Throughout the article I've made that adjustment with spreadsheets first then our R/tidyverse comparison